assets/css/app/overrides.scss

```diff
@@ -37,13 +37,3 @@ $breadcrumb-span-color: hsl(0, 0%, 13%);
 }
 
 
-#wrapper.main {
```

Comment:
Without this, what's the right way to control the width of the body content? It looks like the body is wider than the header.

Reply:
The body should be the exact same as the header as they are both inside Bootstrap containers. But you can go down the route of modifying the _sass/bootstrap/_variables.scss partial and changing the default container widths there.

Reply:
After 1100 pixels or so this happens
![screenshot from 2018-03-09 10-11-08](https://user-images.githubusercontent.com/409011/37217101-3bf70d62-2382-11e8-8ad3-1a2efb88d167.png)

Reply:
Sorry @danrue, this is because of additional padding on the navbar-brand element.

![navbar-brand-padding](https://user-images.githubusercontent.com/4564433/37217401-532ce3ac-23b5-11e8-9823-178ad8dd49f1.png)

I will update the theme now so that doesn't happen :)